### PR TITLE
Added quotes to fix strings ending with :

### PR DESCRIPTION
**Pull request by pfwd**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

Added quotes around English translation strings that ended in : as these were breaking yaml parsers and IDES such as netbeans

### CODE REVIEW

**Comment by stloyd** [comment]
IMO all those `:` should be removed from translations.

**Comment by pfwd** [comment]
No problem. I can remove them if you want?

**Comment by pfwd** [comment]
@stloyd I've removed them

File: src/Sylius/Bundle/WebBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -575,7 +575,7 @@ sylius:
         state: State
         immutable_warning: Already shipped orders are immutable.
         promotion_coupons: Coupons
-        no_promotion: No coupons were used.order:
+        no_promotion: No coupons were used.order
```
Comment (stloyd) [inline src/Sylius/Bundle/WebBundle/Resources/translations/messages.en.yml]: This one looks like wrong merge, IMO should be `No coupons were used.`

@pjedrzejewski @michalmarcinkowski ping

**Comment by pjedrzejewski** [comment]
Thanks @pfwd Peter for raising this! We are currently working on keys transformer, which will simplify all keys in Sylius (so far 1393 -> 796), but preserve translations and we will remove all ":" automatically when running this transformation. :) I will leave this open so that we keep the track of this problem and if we discover any problem with the solution.

**Comment by pfwd** [comment]
OK sounds good. Thanks for looking into it :+1:

**Comment by michalmarcinkowski** [comment]
Fixed in #3908.

**Pull request closed without merging**